tmux/demos/medical/BBN-integrate-Kitware-R18.yml

```diff
@@ -0,0 +1,116 @@
+#
+# System configuration to run the ANGEL system for the BBN system.
+#
+# This configuration is for the M2 tourniquet task.
+#
+
+name: BBN-R18-Chest-Seal
+root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
+
+# Optional tmux socket
+# socket_name: foo
+
+# Note that the pre and post options have been deprecated and will be replaced by
+# project hooks.
+
+# Project hooks
+
+# Runs on project start, always
+# on_project_start: command
+on_project_start: |
+  export ROS_NAMESPACE=${ROS_NAMESPACE:-/kitware}
+  export HL2SS_API_URL=${HL2SS_API_URL:-128.33.193.178:8000}
+  export BBN_OUTPUT_URL=${BBN_OUTPUT_URL:-tcp://128.33.193.178:5557}
+  export CONFIG_DIR=${ANGEL_WORKSPACE_DIR}/config
+  export NODE_CONFIG_DIR=${ANGEL_WORKSPACE_DIR}/src/angel_system_nodes/configs
+  export MODEL_DIR=${ANGEL_WORKSPACE_DIR}/model_files
+# Run on project start, the first time
+# on_project_first_start: command
+
+# Run on project start, after the first time
+# on_project_restart: command
+
+# Run on project exit ( detaching from tmux session )
+# on_project_exit: command
+
+# Run on project stop
+# on_project_stop: command
+
+# Runs in each window and pane before window/pane specific commands. Useful for setting up interpreter versions.
+# pre_window: rbenv shell 2.0.0-p247
+
+# Pass command line options to tmux. Useful for specifying a different tmux.conf.
+# tmux_options: -f ~/.tmux.mac.conf
+tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
+
+windows:
+  # Connection with the HL2 app via HL2SS sensor data streaming and the
+  # ROS-TCP-Endpoint
+  - sensor_input: ros2 run angel_system_nodes redis_ros_bridge --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p url:=${HL2SS_API_URL}
+      -p image_topic:=\image_0
+      -p hand_pose_topic:=HandJointPoseData
+
+  - run_image_timestamp: ros2 run angel_system_nodes image_timestamp_relay --ros-args 
+      -r __ns:=${ROS_NAMESPACE}
+      -p image_topic:=\image_0 
+      -p output_topic:=\image_0_ts
+
+  - pose_estimation: ros2 run angel_system_nodes pose_estimator --ros-args 
+      -r __ns:=${ROS_NAMESPACE}
+      -p image_topic:=\image_0 
+      -p det_topic:=pose_dets 
+      -p pose_topic:=PatientPose 
+      -p det_net_checkpoint:=${ANGEL_WORKSPACE_DIR}/model_files/models/pose_det_model.pth 
+      -p pose_net_checkpoint:=${ANGEL_WORKSPACE_DIR}/model_files/models/pose_model.pth 
```

Comment:
Can you add all the models we are currently using to ansible?

Reply:
they are available at data.kitware.com
DARPA PTG > angel system files > activity classifiers > activity classification from object detections > dev_model_files

Reply:
yes but ansible will be a lot easier to pick up testing the system. So I can just call `ansible pull` and all the files will land exactly where they need to be for the tmux configs to run instead of manually downloading and hoping I grab the correct files and replicate the `models_files` dir. Plus, It'll make getting the models on the BBN system a lot easier too.